src/shared/utils/addAnnotationToCSV.ts

```diff
@@ -0,0 +1,30 @@
+// Annotation row format:
+// https://docs.influxdata.com/influxdb/cloud/reference/syntax/annotated-csv
+export const addAnnotationToCSV = (csv: string): string => {
+  if (!csv) {
+    return csv
+  }
```

Comment:
What is the point of this check to determine if `csv` is falsy? Are we implicitly trying to check for an empty string `''`, or some other value? And if so, why are we returning the falsy value, instead of a fixed value (like `''`)?

Reply:
Good point! Updated